View/DataContexts/DataContext.cs

```diff
@@ -24,6 +24,8 @@ public static DataContext Instance
 
         public event PropertyChangedEventHandler PropertyChanged;
 
+        public Song CurrentSong => AudioPlayer.Instance.CurrentSong;
+        public string PathToImage => CurrentSong == null ? "" : CurrentSong.PathToImage == null ? "" : FileCache.Instance.GetFile(CurrentSong.PathToImage);
```

Comment:
willen we geen fallback image?
mocht een song geen foto hebben zegmaar?

Reply:
lijkt me slim plan